Hmwk#5_JSON/convert.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import csv
+import os
+import sys
+import json
+from typing import List, Union
+
+#--csvfilepath    -f    path to file needed to read
+#--jsonfile   -f    path to file to write
+
+
+
+
+def getArgs(argv = None): 
+    parser = argparse.ArgumentParser(description='Add filepath to file to read from.') #add arguments
+    parser.add_argument('-cf','--csvfilepath', type=str,help='path to file needed to read')
+    parser.add_argument('-jf','--jsonfile', type=str,help='path to file to write')
```

Comment:
If you write "**-cf**" or "**--jf**", it means these arguments are optional, and your program can work without it.

Use a positional argument if it's required.